conf/configuration.go

```diff
@@ -11,11 +13,12 @@ import (
 // Default configuration values.
 // For individual descriptions see Config.
 const (
-	defaultAccountsHost = "10.10.10.70"
-	defaultAccountsPort = "3000"
-	defaultLogLevel     = "info"
-	defaultSiaAPIHost   = "10.10.10.10"
-	defaultSiaAPIPort   = "9980"
+	defaultAccountsHost       = "10.10.10.70"
+	defaultAccountsPort       = "3000"
+	defaultLogLevel           = "info"
+	defaultSiaAPIHost         = "10.10.10.10"
+	defaultSiaAPIPort         = "9980"
+	defaultMinNumberOfPinners = 1
```

Comment:
I think this is fine to leave here, but for our prod servers I think we should make this 2, so that when a server goes down, we have a buffer, and aren't racing against the clock to pin all of its skylinks.

Reply:
Should I hardcode it to 2 for the moment, until we've discussed and implemented https://linear.app/skynetlabs/issue/SKY-279/cluster-wide-setting-for-underpinned-threshold ?

I'm waiting for your input on that one. My view is that the DB is the best place to store that setting (because it's cluster-wide) but we can discuss it under the right issue, either on Liner or on GitHub.

Reply:
We currently have the per-server `PINNER_MIN_PINNERS` setting but that needs to be removed. If we raise it on a single server, that server will try to pin **all** files in the DB and we definitely don't want that.

Reply:
>  In that case we will have 2 months to pin the files on a different server right?
Our renew window on our server is 1 month.

```
user@eu-fin-3:~$ docker exec sia siac renter allowance
Allowance:
  Amount:               144 KS
  Period:               8640 blocks
  Renew Window:         4032 blocks
  Hosts:                50

```
So we should be 1 month to pin data, but that assumes the file is health and on hosts that are health and online for that entire month.

With the recent lost of files I think we need to increase our level of redundancy and assume the worst of our servers and hosts. 

I think from a business standpoint we should not be OK only having 1 server pinning a file.

Reply:
> I'm waiting for your input on that one. My view is that the DB is the best place to store that setting (because it's cluster-wide) but we can discuss it under the right issue, either on Liner or on GitHub.

Responded.  Reiterating here.

We already store cluster wide configs in the `.env` file. Both `common.yml` and `cluster-<cluter-id>.yml` files in last pass are cluster wide. So keeping cluster wide variables hardcoded and non configurable would be deviating from our current approach.

Reply:
to be clear, I don't view this as blocking. we can follow up offline.

Reply:
Resolving this. We'll continue the discussion on Linear and we'll follow up.